app-backend/backsplash/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:8-jre-alpine
+
+ENV ASPECTJ_WEAVER_VERSION 1.8.10
```

Comment:
If Kamon is being removed, we can get rid of the AspectJ references here.

Reply:
Ditto.